Makefile

```diff
@@ -1,7 +0,0 @@
-# This Makefile requires an internal tool. Sorry.
-PACKAGE = getconf
-
-include $(shell makefile-path python.mk)
-
-release:
-	fullrelease
```

Comment:
I strongly suggest keeping a minimal `Makefile` including `make test`, `make release`, `make docs`, for simpler instructions. I can provide reasonable examples if need be.

Reply:
I am not convinced that a Makefile is useful in this case (where commands are simple enough). My main point is this: I am pretty sure that Python developers don't have the habit of running `make update` or `make test` to install dependencies or run tests. They will look at the concise "Development" page of the documentation and see the commands they need. Having `make test` or `nosetests` won't change much: they are equally easy to type.

Also, `make test` is a very limited way to run tests. How do you pass options to the test runner? Sure we could write a Makefile rule like this:

```Makefile
test:
    nosetests ${ARGS}
```

... and document that one can pass options to the test runner by typing `make test ARGS='--stop a.test:TestCase'`. That's not very friendly, to say the least. :) And since test runner options are usually dependent on the test runner, hiding it behind `make test` does not make much sense: to use the right options, the developer must known which test runner the project uses.

Reply:
2 other "top level" packages I use have makefiles:
- https://github.com/PyCQA/pylint/blob/master/Makefile
- https://github.com/ytdl-org/youtube-dl/blob/master/Makefile

Neither are mandatory to properly use said packages but I guess they serve as a sort of "live documentation" of how a package works, is tested, is installed, etc.

However from my POV, a "working on/with `getconf`" section in the docs would be fine as well. Your call :)

Reply:
All right, then! :) I have kept a minimal Makefile.